documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -31,7 +31,7 @@ NOTE: The above links refers to the nightly snapshot build of the {prodname} mas
 If you are looking for a non-snapshot version, please select the appropriate version of this documentation in the menu to the right.
 endif::[]
 ifeval::['{page-version}' != 'master']
-* https://repo1.maven.org/maven2/io/debezium/debezium-server-distro/{debezium-version}/debezium-server-distro-{debezium-version}-distribution.tar.gz[{prodname} Server distribution]
+* https://repo1.maven.org/maven2/io/debezium/debezium-server-distro/{debezium-version}/debezium-server-distro-{debezium-version}.tar.gz[{prodname} Server distribution]
```

Comment:
Ok, but didn't we want to remove "distro" also from the artifact id? I.e. https://repo1.maven.org/maven2/io/debezium/debezium-server/{debezium-version}/debezium-server-{debezium-version}.tar.gz

Reply:
That would mean moving the assmebly at the level of parent project which i found impossible to get running so I had to introduce the distro module in the first place. And as you can see wildlfy has also `dist` in the artifact name https://mvnrepository.com/artifact/org.wildfly/wildfly-dist

Reply:
Oook. Note that's why I suggested to rename the modules; but let's follow WildFly's example then. "distro" -> "dist"?

Reply:
@gunnarmorling Done

Reply:
Sweet, thx!